legacy/tensormap-server/app/resources/model_templates/keras_temp_multi_classification.py

```diff
@@ -1,27 +1,27 @@
+import numpy as np
+import pandas as pd
 import tensorflow as tf
+from sklearn.metrics import accuracy_score, f1_score, precision_score, recall_score
+from sklearn.model_selection import train_test_split
 from tensorflow import keras
 from tensorflow.keras import layers
-import numpy as np
-from sklearn.metrics import f1_score, recall_score, precision_score, accuracy_score
-from sklearn.model_selection import train_test_split
-import pandas as pd
 
 np.random.seed(0)
 
 dataCsv = pd.read_csv()
-        
-_x = dataCsv[]
-_y = dataCsv[]
 
-x_train, y_train, x_test, y_test = train_test_split(_x, _y, random_state=42, shuffle=True, test_size=)
+_x = dataCsv[0]
+_y = dataCsv[1]
+
+x_train, y_train, x_test, y_test = train_test_split(_x, _y, random_state=42, shuffle=True, test_size=0.10)
 
-network = tf.keras.models.Sequential(name='userModel')
+network = tf.keras.models.Sequential(name="userModel")
 
-network.compile (optimizer = ,loss = )
+network.compile(optimizer="adam", loss="Cross-entropy loss")
```

Comment:
Don't hard code optimizers and loss function in here. Create a variable on top of the file and assign the variable

Reply:
it's done

Reply:
it's done

Reply:
it's done